cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/loadbalancer/GcpLoadBalancerScheme.java

```diff
@@ -32,4 +33,20 @@ public static GcpLoadBalancerScheme getScheme(CloudLoadBalancer cloudLoadBalance
                 return INTERNAL;
         }
     }
+
+    public static GcpLoadBalancerScheme getScheme(String gcpType) {
+        switch (gcpType) {
+            case Constants.EXTERNAL_STRING:
```

Comment:
Having a static class of constants in an enum class, which is already a class of constants, just feels odd. Could you not do something like:
```
if (INTERNAL.gcpType.equals(gcpType)) {
    return INTERNAL; 
} else etc...
```
That way you wouldn't need to reference the "INTERNAL" and "EXTERNAL" strings explicitly in two different places.